README.rst

```diff
@@ -1592,6 +1592,7 @@ sample_pki
         # url(r'^geo/', include((get_geo_urls(geo_views), 'geo'), namespace='geo'),),
 
         # openwisp-controller urls
+        url(r'', include(('openwisp_controller.config.urls', 'config'),namespace='config')),
```

Comment:
Needed to make this change but able to avoid adding the url in `openwisp_controller/urls.py`'s `urlpatterns`! :smile:

Reply:
there's a missing space after the last comma